internal/util/tools/docs/docs.go

```diff
@@ -92,8 +93,12 @@ func findJsonSections(fileLines []string) [][]string {
 		if inASection {
 			currentSection = append(currentSection, line)
 		}
-		if line == "```"+infoString {
+		if line == "```json" && expectingSection {
 			inASection = true
+			expectingSection = false
```

Comment:
Optional: it'd be cleaner to have an explicit state machine with states: EXPECTING_SECTION, IN_SECTION, NOT_IN_SECTION.